telemetry/scripts/backup_crontabs

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This script backs up all crontabs on a computer to a specified directory
+# Crontabs are read from /var/spool/cron/tabs/
+# Must be run as root user
+# Suggested usage: Run periodically on root user's crontab
+
+readonly BACKUP_DIR="/data/crontab_backup"
+
+date --utc
+
+# Make backup dir, root will be owner
+mkdir -p $BACKUP_DIR
+
+for user_crontab in /var/spool/cron/tabs/*; do
+    [[ -e $user_crontab ]] || continue  # exit if no crontabs found
+    echo "Backing up crontab for $(basename $user_crontab)..."
+    backup_file="${BACKUP_DIR}/$(basename $user_crontab)-$(date --utc +"%Y%m%d")"
+    cp --verbose $user_crontab $backup_file
+    chmod 664 $backup_file  # Give read permissions to all users  
```

Comment:
Could also change the owner to one of the standard users (edtk likely) instead of root

Reply:
I think you updated the crontab google sheet but just making a note here for others:  that should be done too

Reply:
I left it as root user due to the difference in standard users across the computers. Some of the CD and DDS computers have uid `1000` as `transfer`, while some have `1000` as `dds`. As this is how the NAS is set up though, it is probably appropriate.

I'll change this to follow your suggestion, we can revert it later if we change our mind.